content/javascript/concepts/arrays/terms/length/length.md

```diff
@@ -69,3 +69,26 @@ colors.length = 3;
 console.log(colors);
 // Output: ['purple', 'orange', 'yellow']
 ```
+
+## Codebyte Example
+
+The example below defines a new array `groceries` and then logs the four array items to the console. Next, to delete the last item in the array, its `length` property is set to 3. Finally, to confirm the last item has been removed, the updated array and the value of item 4 (`undefined`) are logged to the console.
+
+```codebyte/javascript
+const groceries = ['Eggs', 'Cheese', 'Bread', 'Broccoli'];
+
+console.log(`\nCurrent grocery list:\n`);
+groceries.forEach((item, i) => {
+    console.log(`${i + 1}) ${item}`);
+});
+console.log(`\nTotal grocery items: ${groceries.length}`); // 5
```

Comment:
```suggestion
console.log(`\nTotal grocery items: ${groceries.length}`); 
```

Reply:
you can remove the commented output because it will show the appropriate values when the user runs the code